src/backend/joanie/core/api/client.py

```diff
@@ -690,6 +690,15 @@ class CourseViewSet(
 
     GET /api/courses/:<course_id>
         Return one course if an id is provided.
+
+    GET /api/courses/:<course_id>/wish
+        Return wish status on this course for the authenticated user
```

Comment:
Richie will not be aware to the course id. We should use course code instead ...

Reply:
It seems to be a good trade off indeed.